components/skus/browser/rs/lib/src/sdk/credentials/mod.rs

```diff
@@ -49,9 +49,10 @@ where
                     if let Some(expires_at) = expires_at {
                         // attempt to refresh credentials if we're within 5 days of expiry
                         if Utc::now().naive_utc() > (expires_at - chrono::Duration::days(5)) {
+                            error!("Within 5 days of expiry; refreshing order credentials.");
                             let refreshed = self.refresh_order_credentials(order_id).await;
                             if refreshed.is_err() {
-                                continue;
+                                error!("Error refreshing order credentials.");
                             }
                         }
```

Comment:
looks like we have this identical code in 2 locations (just highlighting for later)